node/coinstacks/bitcoin/api/src/app.ts

```diff
@@ -43,8 +50,27 @@ app.get('/', async (_, res) => {
 app.use(middleware.errorHandler)
 app.use(middleware.notFoundHandler)
 
-const server = app.listen(port, () => logger.info('Server started'))
+const registry = new Registry()
+  .formatAddress((address) => {
+    const decoded = bech32.decodeUnsafe(address.toLowerCase())
+    if (decoded?.prefix === 'bc') return address.toLowerCase()
```

Comment:
bech32 addresses are case insensitive and the goal of formatting addresses is just to ensure we are treating all addresses the same way to ensure proper string matching. I didn't want to lowercase everything since non bech32 btc addresses are case sensitive and I don't know of an easy way to re-checksum them if needed

Reply:
Export format function for use in restful tx history and unchained-client tx parsing to ensure we are speaking the same address format throughout the stack.